apps/managedidentity/managedidentity.go

```diff
@@ -46,6 +49,9 @@ const (
 	// IMDS
 	imdsEndpoint   = "http://169.254.169.254/metadata/identity/oauth2/token"
 	imdsAPIVersion = "2018-02-01"
+
+	//system assigned managed identity
```

Comment:
I'd delete this as redundant but if you want a comment, please put a space after the `//` to avoid linter complaints